src/test/java/de/retest/recheck/review/ignore/matcher/ElementClassMatcherTest.java

```diff
@@ -13,62 +13,86 @@
 class ElementClassMatcherTest {
 
 	Element element;
-	ElementClassMatcher cut;
+	ElementClassMatcher matcher;
```

Comment:
We always use `cut` to refer to the _class under test_ (see wiki), could you please revert this?